pyfar/dsp/filter/band_filter.py

```diff
@@ -450,3 +450,68 @@ def crossover(signal, N, frequency, sampling_rate=None):
         # return the filtered signal
         signal_filt = filt.process(signal)
         return signal_filt
+
+
+def notch(signal, center_frequency, quality, sampling_rate=None):
+    """
+    Create and apply a second order IIR notch filter.
+
+    A notch filter has a magnitude of 0 (-infinity dB) at the center frequency
+    and a magnitude of 1 (0 dB) at almost all other frequencies. This is a
```

Comment:
Why don't we just use the same definition as scipy? It's fits better than this one to be honest.
```suggestion
    A notch filter is a band-stop filter with a narrow bandwidth (high quality factor). It rejects a narrow frequency band and leaves the rest of the spectrum little changed. This is a
```

Reply:
I found it more illustrative if also mentioning the gains at and away from the center frequency. How about this, which is closer to the scipy docs?